frontend/src/v5/ui/controls/chip/chip.component.tsx

```diff
@@ -20,6 +20,6 @@ import { COLOR } from '../../themes/theme';
 import { ChipBase, ChipType } from './chip.styles';
 
 export const Chip = forwardRef((props: ChipType, ref: any) => {
-	const { color = COLOR.PRIMARY_MAIN, ...otherProps } = props;
-	return <ChipBase $coloroverride={color} {...otherProps} ref={ref} />;
+	const { color = COLOR.PRIMARY_MAIN, variant, ...otherProps } = props;
+	return <ChipBase $coloroverride={color} $variant={variant} {...otherProps} ref={ref} />;
```

Comment:
**This is not a change request**
I've noted you need the ref only for `PriorityLevelChip` as the first child of the tooltip must be capable to hold a ref.

Another approach could have been to wrap the Chip in a div (which is by default capable to hold a ref) in `PriorityLevelChip` without using the `forwardRef` here. But again, this is not a change request